flang/lib/Semantics/symbol.cpp

```diff
@@ -246,7 +246,7 @@ void GenericDetails::CopyFrom(const GenericDetails &from) {
 // This is primarily for debugging.
 std::string DetailsToString(const Details &details) {
   return common::visit(
-      common::visitors{
+      common::visitors{//
```

Comment:
Stray "//" ?

Reply:
No idea how that happened. I don't use VI. :)